pkg/scheduler/core/generic_scheduler.go

```diff
@@ -208,6 +208,7 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 		return result, err
 	}
 
+	klog.V(2).Infof("DEBUG: Compute predicates pod: %v/%v/%v", pod.Tenant, pod.Namespace, pod.Name)
```

Comment:
I thought those predicates for density test will be same for all pods

Reply:
predicates are scheduler scope. this log is log for the pod name.